addons/event_sale/models/event_registration.py

```diff
@@ -65,20 +86,20 @@ def _prepare_attendee_values(self, registration):
             })
         return att_data
 
-    def summary(self):
-        res = super(EventRegistration, self).summary()
-        if self.event_ticket_id.product_id.image_128:
-            res['image'] = '/web/image/product.product/%s/image_128' % self.event_ticket_id.product_id.id
-        information = res.setdefault('information', {})
-        information.append((_('Name'), self.name))
+    def _mark_as_paid(self):
```

Comment:
_action_set_paid ? That way we are synchronized with other action method on registration model.